packages/template-retail-react-app/package.json

```diff
@@ -96,7 +96,7 @@
     },
     {
       "path": "build/vendor.js",
-      "maxSize": "320 kB"
+      "maxSize": "340 kB"
```

Comment:
As mentioned in a previous comment, we can reduce this maxSize closer to 320 kB, after changing how we import `isEqual` from lodash.